pokete_data/trainers.py

```diff
@@ -301,14 +301,59 @@
                  [" < NOOOOOO; WHYYY ARE YOU BETTER ??!!!1!!!111"], 69, 61)
         },
     ],
+    "playmap_39": [{
+        "poke": ("megapois", 2000),
+        "args": ("Angela the gardener", "She",
+                 [" < Have you allready fought our lead trainer?", " < If so, look at cool Pokete!"],
+                 [" < You're a loooser!"],
+                 [" < Cool, huh?"], 8, 37)
+        },
+    ],
     "playmap_40": [{
         "poke": ("rustacean", 1800),
         "args": ("Gustav the angler", "He",
                  [" < My Pokete will blow you away!"],
-                 [" < You're a fucking loooser!"],
+                 [" < You're a loooser!"],
                  [" < Cool, huh?"], 58, 12)
         },
-    ]
+    ],
+    "playmap_41": [{
+        "poke": ("wheeto", 2000),
+        "args": ("Daniel the farmer", "He",
+                 [" < Howdi mate!", " < How are ya doin?", " < My Pokete is one of its' kind", " < The good stuff!"],
```

Comment:
```suggestion
                 [" < Howdi mate!", " < How are ya doin?",
                  " < My Pokete is one of its kind",
                  " < The good stuff!"],
```

or even more "slangy":

```suggestion
                 [" < Howdi mate!", " < How ya doin?",
                  " < My Pokete is one of its kind",
                  " < The good stuff!"],
```